import java.io.FileNotFoundException;
import java.io.FileOutputStream;
import java.io.IOException;

/*This writes objects to the file*/
public class WriteObjects {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		System.out.println("Writing Objects ..");
		Person mike = new Person(543, "Mike");
		Person sue = new Person(123, "Sue");
		System.out.println(mike);
		System.out.println(sue);
		//write the objects in a binary file.
		/*Whatever I did here to catch the exception is called try with
		 * resources. so try would automatically call the close method of 
		 * fs object*/
		try(FileOutputStream fs = new FileOutputStream("people.bin")){
			
		} catch (FileNotFoundException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		} catch (IOException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
		
	}

}
